tests/functional/conftest.py

```diff
@@ -85,29 +87,29 @@ def prepare_speculos_args(firmware: Firmware, display: bool):
 # Depending on the "--backend" option value, a different backend is
 # instantiated, and the tests will either run on Speculos or on a physical
 # device depending on the backend
-def create_backend(backend: str, firmware: Firmware, display: bool):
-    if backend.lower() == "ledgercomm":
+def create_backend(backend_name: str, firmware: Firmware, display: bool):
```

Comment:
You can add the newest fixture from the ragger template

Reply:
done